srml/treasury/src/lib.rs

```diff
@@ -233,6 +233,15 @@ decl_storage! {
 		/// Proposal indices that have been approved but not yet awarded.
 		Approvals get(fn approvals): Vec<ProposalIndex>;
 	}
+	add_extra_genesis {
+		build(|_config| {
+			// Create Treasury account
+			let _ = T::Currency::make_free_balance_be(
+				&MODULE_ID.into_account(),
```

Comment:
`&Self::accountid()` doesn't work here right?

Reply:
no actually inside build function self is GenesisConfig

Reply:
`Module<T>::account_id()` should work

Reply:
oh yes indeed bette. done.